Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerOsInformation.ps1

```diff
@@ -482,4 +482,23 @@ function Invoke-AnalyzerOsInformation {
         Write-Verbose "No Windows Features where collected. Unable to process."
     }
     # cSpell:enable
+
+    if ($null -ne $osInformation.EventLogInformation) {
+        $days = 7
+        $testDate = (Get-Date).AddDays(-$days)
+        foreach ($logType in $osInformation.EventLogInformation.Keys) {
+            $logInfo = $osInformation.EventLogInformation[$logType]
+            # If the log isn't at the max limit, it is possible that they just cleared the logs or on a new server. This should be rare scenario.
+            if ($logInfo.LastLogEntry -gt $testDate -and
+                $logInfo.FileSize -ge $logInfo.MaxSize) {
+                $params = $baseParams + @{
+                    Name             = "Event Log - $logType"
+                    Details          = "--ERROR-- Not enough logs to cover $days days. Last log entry is at $($logInfo.LastLogEntry)." +
```

Comment:
Should we make this a `Warning` instead of an `Error` as it doesn't cause any issue (but makes RCA harder in case of an issue).

Reply:
For now, let's leave it as an `Error` till we get feedback from it. In most cases, you should have more than 7 days' worth of application logs.